server/health/healthchecks.js

```diff
@@ -0,0 +1,25 @@
+const postgresConfig = require('../config').postgresConfig
+const {addCriticalTest, runTests} = require('fh-health')
```

Comment:
Do we really need this library? Looked through the library and it's good but it just seems unnecessary

A much simpler and similar alternative would be something like this:

```javascript
// health/index.js

module.exports = function Health(db) {
  
  function databaseConnectivityCheck() {
    return new Promise((resolve, reject) => {
      db.sequelize.authenticate()
        .then(resolve)
        .catch((err) => {
          console.error(err) // log the actual error
          reject('Database Connection Error') // return a generic one
        })
    })
  }

  function runChecks() {
    const checks = [ databaseConnectivityCheck() ] // can add more if we need
    return Promise.all(checks)
  }

  return runChecks
}
```

and then in `server.js`

```javascript
const runHealthChecks = require('./lib/health')(models)

app.get('/healthz', async (req, res) => {
  try {
    await runHealthChecks()
    res.json({ok: true})
  } catch(error) {
    res.status(503)
    res.json({ error })
  }
})
```

I feel like this is a simpler way to go but still allows us to achieve a similar thing as `fh-health.`